docs/partial_source/design/building_blocks.rst

```diff
@@ -376,6 +376,9 @@ First, let's set our desired backend as torch.
    :align: center
    :width: 75%
 
+In this example, we observe that both the Ivy code, and the native backend code, compile to the same desired backend code, which in our example was torch.
```

Comment:
"Here we should an example where `ivy` code, raw `torch` code, and also a mixture of `ivy` and `torch` code all compile to the same `torch` computation graph.